docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>Nice touch using idata ;)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>IIUC, this will only generate posterior predictive samples for the first 500 draws of the first chain. Could <code>samples</code>  be removed to use the default? </p>

Reply:
<p>yes. typical example for a chain &amp; draw unbiased random thinning convenience method</p>